templates/patroni.yml.j2

```diff
@@ -61,6 +61,10 @@ bootstrap:
 bypass_api_service: true
 log:
   dir: /var/log/postgresql
+  dateformat: "%Y-%m-%d %H:%M:%S %Z"
+  format: "%(asctime)s [%(process)d]: %(levelname)s: %(message)s "
+  file_num: 10
+  file_size: 10485760
```

Comment:
We can probably reduce this to 600 bytes, so it should rotate the logs close to every minute (with the default app name - `postgresql-k8s` - most part of the messages have 91 bytes on primary and 121 bytes on replica, and they are sent to the log file every 10 seconds - according to the value of the `loop_wait` parameter). It's not an ideal solution, but it's better to make it rotate the logs more often.

Reply:
It smells a bit, but up to @7annaba3l as a feature requester.

Reply:
Hi @dragomirp @taurus-forever @marceloneppel 
Let's go for the proposal from @marceloneppel . 
We will revisit this in the future and probably cleanup and align the config to the server part using logrotate ...